frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/vm-wizard-virtualhardware-row.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Kebab, KebabOption } from '@console/internal/components/utils';
+import { VMWizardStorageType, VMWizardStorageWithWrappers } from '../../types';
+import { CDSimpleRow } from '../../../vm-disks/cd-row';
+import {
+  VMWizardStorageBundle,
+  VMWizardStorageRowActionOpts,
+  VMWizardStorageRowCustomData,
+} from '../storage-tab/types';
+import { vmWizardStorageModalEnhanced } from '../storage-tab/vm-wizard-storage-modal-enhanced';
+
+const menuActionEdit = (
+  storageWithWrappers: VMWizardStorageWithWrappers,
+  { wizardReduxID, withProgress }: VMWizardStorageRowActionOpts,
+): KebabOption => {
+  return {
+    label: 'Edit',
+    isDisabled: [
+      VMWizardStorageType.WINDOWS_GUEST_TOOLS,
+      VMWizardStorageType.WINDOWS_GUEST_TOOLS_TEMPLATE,
+    ].includes(storageWithWrappers.type),
+    callback: () =>
+      withProgress(
+        vmWizardStorageModalEnhanced({
+          blocking: true,
+          wizardReduxID,
+          storage: storageWithWrappers,
+        }).result,
+      ),
+  };
+};
+
+const menuActionRemove = (
+  { id, type }: VMWizardStorageWithWrappers,
+  { withProgress, removeStorage }: VMWizardStorageRowActionOpts,
+): KebabOption => ({
+  label: 'Delete',
+  isDisabled: [
+    VMWizardStorageType.PROVISION_SOURCE_DISK,
+    VMWizardStorageType.PROVISION_SOURCE_TEMPLATE_DISK,
+    VMWizardStorageType.V2V_VMWARE_IMPORT_TEMP,
+  ].includes(type),
+  callback: () =>
+    withProgress(
+      new Promise((resolve) => {
+        removeStorage(id);
+        resolve();
+      }),
+    ),
+});
+
+const getActions = (
```

Comment:
these actions seem to be the same as the one in vm-wizard-storage-row. Can we export them from there  and delete them here?